agent/config/runtime.go

```diff
@@ -1414,6 +1414,16 @@ type RuntimeConfig struct {
 	//
 	Watches []map[string]interface{}
 
+	// XDSRebalanceDrainRateLimit and XDSRebalanceDrainMaxBurst control the rate
+	// at which xDS sessions will be terminated to rebalance load between servers,
+	// using a token bucket rate limiter.
+	//
+	// See https://en.wikipedia.org/wiki/Token_bucket for more about token buckets.
+	//
+	// hcl: xds { rebalance_drain_rate = (float64|MaxFloat64) rebalance_drain_burst = int }
+	XDSRebalanceDrainRateLimit rate.Limit
+	XDSRebalanceDrainMaxBurst  int
```

Comment:
Having talked this through a bit on Slack, we're going to remove these user-facing options and scale the rate limit dynamically based on number of proxies.